nfd/node-feature-discovery-openshift.yaml

```diff
@@ -12,8 +12,8 @@ spec:
   resourceLabels:
     - sgx.intel.com/epc
   operand:
-    image: quay.io/openshift/origin-node-feature-discovery:4.13
+    image: quay.io/openshift/origin-node-feature-discovery:4.14
     imagePullPolicy: Always
     servicePort: 12000
   workerConfig:
-    configData: |
+    configData: |
```

Comment:
Slight difference with no newline, rest LGTM

Reply:
do we need the new line?